src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -438,7 +438,13 @@ export const QueryResultPane = () => {
                     renderGridPanel()}
                 {metadata.tabStates!.resultPaneTab ===
                     qr.QueryResultPaneTabs.Messages && (
-                    <div className={classes.messagesContainer}>
+                    <div
+                        className={classes.messagesContainer}
+                        data-vscode-context={JSON.stringify({
```

Comment:
Can you make a helper function for this?

Reply:
I couldn't find a good way to make a helper function for this, it's just stringifying an object. We can look into this if we have more usage of this attribute.